pubspec.yaml

```diff
@@ -30,6 +31,7 @@ flutter:
   assets:
     - assets/images/
     - assets/images/icon/
+    - assets/text/
```

Comment:
Is it just my GitHub diff view adding the extra leading indent?

Reply:
Perhaps, but it had a merge conflict anyway so it's changed now.